test/models/erc1155-ownable.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { defaultWeb3Connection, getPrivateKeyFromFile } from '../utils';
+import { ERC1155Ownable, Web3Connection } from '../../src';
+import { expect } from 'chai';
+import { Account } from 'web3-core';
+
+describe(`ERC1155 Ownable`, () => {
+  let contract: ERC1155Ownable;
+  let web3Connection: Web3Connection;
+  let contractAddress: string;
+  let bob: Account;
+  const initial = {
+    uri: 'https://my.token.uri',
+  };
+  before(async () => {
+    web3Connection = await defaultWeb3Connection(true, true);
+    bob = web3Connection.Web3.eth.accounts.privateKeyToAccount(
+      getPrivateKeyFromFile(1)
+    );
+  });
+
+  it(`Deploys the contract`, async () => {
+    const deployer = new ERC1155Ownable(web3Connection);
+    deployer.loadAbi();
+    const tx = await deployer.deployJsonAbi('https://my.token.uri');
+    expect(tx.blockNumber).to.exist;
+    contractAddress = tx.contractAddress!;
+  });
+
+  describe(`Methods`, () => {
```

Comment:
Tests are missing a `it` where it asserts that no one else can use the methods other than the owner

Reply:
This one will be a little difficult, since the sender address is retrieved inside the `callTx`/`sendTx` using the actual connection and I cannot override the connection since its a read-only property.